runtime/runtime/src/ext.rs

```diff
@@ -92,7 +92,7 @@ impl<'a> RuntimeExt<'a> {
     fn append_action(&mut self, receipt_index: u64, action: Action) {
         self.action_receipts
             .get_mut(receipt_index as usize)
-            .expect("receipt index should be present")
+            .expect("receipt index should be present; qed")
```

Comment:
The idea is to use some idiom in panic messages which would tell the reader that this should never panic. Just to not having to think each time how to express the same message.

Would be nice to have a standard way to express that the panic is not possible. WDYT?

Reply:
Sounds good to me.

Reply:
I don't see the difference between panic that is not possible and panic that is possible. Panic always means violation of invariant. If violation is recoverable then this should not be a panic. Also, I don't think I saw `qed` used anywhere in the code outside Parity. Also, QED means "what was to be shown" https://en.wikipedia.org/wiki/Q.E.D. , which makes sense to be used in the proofs, but when we say `receipt index should be present; qed` it means `receipt index should be present; what was to be shown` which I don't understand how to interpret, because it implies that we were going to show that "receipt index should be present", but we did not show it, because it is not a proof or an argument of some sorts.

Reply:
Ok, we don't need it.

What I wanted to cover the following: `Err` cases which are unreachable. 

What if we agree to use `.expect` only in cases when we sure that API guaranties to hold invariant that `Err` case is unreachable?

Reply:
What is unreachable `Err` case?

Reply:
Like this:
https://github.com/near/borsh/issues/59

The interface can be simplified since since Write implementation for Vec never returns Err